docs/tutorial/pre-requisites-of-running-on-chain-spam-protected-chat2.md

```diff
@@ -85,24 +85,20 @@ We include instructions for [sepolia-faucet.pk910.de](https://sepolia-faucet.pk9
 
    Follow the instructions to register and verify the account.
 
-2. Click on "Create a new API key".
+2. An API Key named "My First Key" should be auto-generated. Click on it, otherwise click on the "Create New API Key" button.
 
-    ![image](https://github.com/waku-org/nwaku/assets/68783915/3cf1d537-9f73-4a6a-b1ef-bbac148bac47)
+    ![image](https://github.com/user-attachments/assets/40cedd01-0282-46f1-a7cd-604bb3f29cae)
```

Comment:
I'm not pretty sure if it's a problem, but it seems the image is now your gh user's content

Reply:
Indeed, it was previously uploaded to the repo which is safer (less chess of disappearing).

Reply:
It's weird, because there's not really an "assets" directory in our repo. If we try to browse to https://github.com/waku-org/nwaku/assets I get a 404.

I can create one, but in both versions we use the same system, like when a file is uploaded to an issue or PR description, GitHub manages the file under the hood without it being part of the repo.

Either the old images were added in a different way (which I haven't been able to figure out) or GitHub changed its asset management system/naming.

I also tried to attach the image in a PR's description and it was also under `user-attachments`

Reply:
WDYT @fryorcraken @NagyZoltanPeter? should I create an assets folder in the repo? or is there a specific way of attaching the files in GH so it appears in a path similar to the previous one?
I googled it but whatever I do, it always ends up under `user-attachments`

Reply:
asset folder :)